dali/operators/math/expressions/arithmetic.h

```diff
@@ -313,6 +313,7 @@ class ArithmeticGenericOp : public Operator<Backend> {
 
   bool SetupImpl(std::vector<OutputDesc> &output_desc, const workspace_t<Backend> &ws) override {
     output_desc.resize(1);
+    auto curr_batch_size = ws.GetRequestedBatchSize(0);
```

Comment:
I guess we will probably plan even more broadcasting here, but this operator is perfectly fine with obtaining batch size from it's inputs right now instead of using the GetRequestedBatchSize.

In this case in currently can have [1, 3] inputs.

Reply:
Still, it's not requested Batch Size. It's an input Batch Size.

Reply:
The assert won't do, we need to calculate common batch size, and as we are passing it to PropagateShapes, make sure it's used there. Or move the calculation to PropagateShapes. The first is easier.